### PR TITLE
Lenses: Fixed livenessProbe to connect to configured port

### PR DESCRIPTION
**Pull request by thall**
This change is [Reviewable](https://reviewable.io/reviews/landoop/kafka-helm-charts/72)

### CODE REVIEW

File: charts/lenses/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 description: A chart for Lenses
 name: lenses
-version: 2.2.14
-appVersion: 2.2.9
+version: 2.2.15
+appVersion: 2.2.10
```
Comment (andrewstevenson) [inline charts/lenses/Chart.yaml]: We don't need to bump the appVersion as we 2.2.9 is latest Lenses release

Comment (thall) [reply charts/lenses/Chart.yaml]: Ok, i was unsure if i needed to change that. I fix.

Comment (thall) [reply charts/lenses/Chart.yaml]: hm doesnt seem to work on linux.
```
nic@ucnd8132rff:~/development/kafka-helm-charts$ ./package.sh 
Cleaning and preparing
rm: missing operand
Try 'rm --help' for more information.
```

**Comment by thall** [comment]
See #73

**Pull request closed without merging**